addons/l10n_ie/data/template/account.tax-ie.csv

```diff
@@ -164,10 +164,10 @@
 ,"","","","","","","","","base","refund","","",""
 ,"","","","","","","","","tax","refund","","","-T2"
 "ie_tax_purchase_eu_goods_0","0% EU G","0% EU Goods","0%","purchase","percent","0","ie_tax_group_vat_0","True","base","invoice","","","+E2"
-,"","","","","","","","","tax","invoice","","","+T1"
+,"","","","","","","","","tax","invoice","-100","","+T1"
```

Comment:
arfff not doing the same as before :-/

wait a minute... the tax is 0% anyway??

Reply:
yeah... I also checked this one with OLMA. This doesn't make any sense since it's a 0% tax

Reply:
constraint to be reviewed after the OXP